Dockerfile

```diff
@@ -0,0 +1,33 @@
+# vim:set ft=dockerfile:
+ARG BASE_IMAGE_TAG
+FROM pavics/workflow-tests:${BASE_IMAGE_TAG:-latest}
```

Comment:
`latest` is whenever someone push something to `master`, which is never tested !  The current version in production is `230601`, this one should be the default.   Hum, how to keep this one up-to-date without additional manual commit on each Jupyter env release ...